CoreAlgorithmPlugins/test/unit/src/au/gov/asd/tac/constellation/plugins/algorithms/sna/centrality/PagerankCentralityPluginNGTest.java

```diff
@@ -102,15 +102,15 @@ public void testNormalisedPagerank() throws Exception {
         final PluginParameters parameters = instance.createParameters();
         parameters.setBooleanValue(PagerankCentralityPlugin.TREAT_UNDIRECTED_BIDIRECTIONAL_PARAMETER_ID, true);
         parameters.setFloatValue(PagerankCentralityPlugin.DAMPING_FACTOR_PARAMETER_ID, 0.85f);
-        parameters.setIntegerValue(PagerankCentralityPlugin.ITERATIONS_PARAMETER_ID, 100);
+        parameters.setIntegerValue(PagerankCentralityPlugin.ITERATIONS_PARAMETER_ID, 2);
         parameters.setFloatValue(PagerankCentralityPlugin.EPSILON_PARAMETER_ID, 1E-8f);
         parameters.setBooleanValue(PagerankCentralityPlugin.NORMALISE_AVAILABLE_PARAMETER_ID, true);
         PluginExecution.withPlugin(instance).withParameters(parameters).executeNow(graph);
 
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId0), 0.22181106f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId1), 0.4309117f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId2), 0.42492065f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId3), 0.8254921f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId4), 1f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId0), 0.18089105f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId1), 0.39393282f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId2), 0.5371851f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId3), 1f);
```

Comment:
@Nova-2119 @cygnus-x-1 so I am OK with updating the unit test but I am worried about this change. Previously `vxId4` was last but not `vxId3` is last. Does that mean the page rank was wrong before or wrong now?

Reply:
It was wrong beforehand. When there is a sink its pagerank is distributed to all other nodes equally. However rather than divide by the maount of nodes - 1 it used to divide by the amount of nodes. This meant that some of the pagerank would be lost and the other nodes got less then they should have.

Reply:
ok thanks @Nova-2119